tests/test_aliases.py

```diff
@@ -37,7 +38,11 @@ class MyModel(BaseModel):
 
         class Config:
             alias_generator = to_upper_case
-            fields = {'my_shiny_field': 'MY_FIELD', 'foo_bar': {'alias': 'FOO'}, 'another_field': {'not_alias': 'a'}}
+            fields = {
+                'my_shiny_field': 'MY_FIELD',
+                'foo_bar': {'alias': 'FOO'},
+                'another_field': {'not_alias': 'a'},
+            }
```

Comment:
I think these are formatting change that seems unrelated to this change. please revert them and all the other formatting changes.